### PR TITLE
feat(content): Added friendly civilian hails

### PR DESCRIPTION
**Pull request by blue-four-seven**
## Summary
Added friendly civilian hails

## Usage examples
{{If your changes affect how game data can be defined, provide examples demonstrating the changes here.}}

## Testing Done
I haven't tested this. I don't know how to, sorry.

### CODE REVIEW

File: data/human/hails.txt [hunk 1]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
 		"uranium or uranium" "uranium"
 		"neodymium and neodymium" "neodymium"
 		"neodymium or neodymium" "neodymium"
-
+phrase "friendly civilian"
```
Comment (bene-dictator) [inline data/human/hails.txt:2710]:
```suggestion

phrase "friendly civilian"
```

File: data/human/hails.txt [hunk 2]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "Longjump"
+        "Glaze"
+        "Deep"
+        "Clink"
+        "Winter"
+        "Solace"
+        "Twinstar"
+        "Sundrinker"
+        "Trinket"
+        "Dancer"
```
Comment (bene-dictator) [inline data/human/hails.txt:2750]: How come these are all Southern Rim planets? I feel like you'd probably find more gambling in the Paradise Worlds or the Core planets.

File: data/human/hails.txt [hunk 3]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+    word
+        " "
+    word
+        "blackjack."
+        "poker."
+        "backgammon."
+        "baccarat."
+        "roulette."
+phrase "friendly civilian"
```
Comment (bene-dictator) [inline data/human/hails.txt:2759]: Please put line breaks between phrases.
```suggestion

phrase "friendly civilian"
```

File: data/human/hails.txt [hunk 4]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "Vague Voyages"
+        "Dark Dimensions"
+        "Mysterious Moons"
+        "Silent Satellites"
+        "Zesty Zeniths"
+        "Daring Discoveries"
+        "Fiery Frontiers"
+        "Global Galaxies"
+        "Rapid Rockets"
+        "Orbital Overload"
```
Comment (bene-dictator) [inline data/human/hails.txt:2780]:
```suggestion
        "Orbital Overload"
        "Stars and Starships"
```
Might as well include one already mentioned in-game!

(Do wish these would be more than just alliterative, e.g. "Southern Rim Spacer News", "Tarazed Today".....)

File: data/human/hails.txt [hunk 5]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "pulse showers are good for your quantum pores."
+        "exposing yourself to neodymium increases your risk of spacepox."
+        "shoddily grown ion algae exhaust your psychic karma."
+        "laser acupuncture let you communicate with nebula ghosts."
+        "star crystals brings luck with long distance relationships."
+        "consuming moon dust accelerates hair growth but causes lunar lunacy."
+        "excessive exposure to zero gravity can lead to phantom limb syndrome."
+        "alien prosthetics often come with unexpected telepathic abilities."
+        "stardate tattoos increase your chances of alien abduction."
+phrase "friendly civilian"
```
Comment (bene-dictator) [inline data/human/hails.txt:2795]:
```suggestion

phrase "friendly civilian"
```

File: data/human/hails.txt [hunk 6]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+         "? It says that"
+    word
+         " "
+    word
+        "pulse showers are good for your quantum pores."
+        "exposing yourself to neodymium increases your risk of spacepox."
+        "shoddily grown ion algae exhaust your psychic karma."
+        "laser acupuncture let you communicate with nebula ghosts."
+        "star crystals brings luck with long distance relationships."
+        "consuming moon dust accelerates hair growth but causes lunar lunacy."
```
Comment (bene-dictator) [inline data/human/hails.txt:2791]:
```suggestion
        "consuming moon dust accelerates hair growth but causes lunacy."
```
"Lunar lunacy", bit tautological, no?

File: data/human/hails.txt [hunk 7]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "Orbital Overload"
+    word
+         "? It says that"
+    word
+         " "
+    word
+        "pulse showers are good for your quantum pores."
+        "exposing yourself to neodymium increases your risk of spacepox."
+        "shoddily grown ion algae exhaust your psychic karma."
+        "laser acupuncture let you communicate with nebula ghosts."
```
Comment (bene-dictator) [inline data/human/hails.txt:2789]:
```suggestion
        "laser acupuncture lets you communicate with nebula ghosts."
```
We don't actually feature any nebulas prominently in human space. Hmm, hopefully the Pirates expansion changes that.

File: data/human/hails.txt [hunk 8]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "Rapid Rockets"
+        "Orbital Overload"
+    word
+         "? It says that"
+    word
+         " "
+    word
+        "pulse showers are good for your quantum pores."
+        "exposing yourself to neodymium increases your risk of spacepox."
+        "shoddily grown ion algae exhaust your psychic karma."
```
Comment (bene-dictator) [inline data/human/hails.txt:2788]:
```suggestion
        "shoddily grown ion algae exhausts your psychic karma."
```
I have no clue what this means but it sounds bad.

File: data/human/hails.txt [hunk 9]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "laser acupuncture let you communicate with nebula ghosts."
+        "star crystals brings luck with long distance relationships."
```
Comment (bene-dictator) [inline data/human/hails.txt:2790]:
```suggestion
        "star crystals bring luck to long distance relationships."
```
Or:
```suggestion
        "star crystals bring luck to interplanetary relationships."
```

File: data/human/hails.txt [hunk 10]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+    word
+        "than"
+        "compared to"
+    word
+        " "
+    word
+        "beam lasers."
+        "electron beams."
+        "heavy lasers."
+phrase "friendly civilian"
```
Comment (bene-dictator) [inline data/human/hails.txt:2815]:
```suggestion

phrase "friendly civilian"
```

File: data/human/hails.txt [hunk 11]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+    word
+        " "
+    word
+        "than"
+        "compared to"
+    word
+        " "
+    word
+        "beam lasers."
+        "electron beams."
```
Comment (bene-dictator) [inline data/human/hails.txt:2813]:
```suggestion
```
Electron beams are locked behind FW so this hail might appear before they are introduced. (Wish we could have more dynamic hails...)

File: data/human/hails.txt [hunk 12]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+    word
+        " "
+    word
+        "stalling engines."
+        "broken batteries."
+        "unstable shielding."
+        "overheating fuel cells."
+        "a glitchy HUD."
+        "faulty ship detection."
+phrase "friendly civilian"
```
Comment (bene-dictator) [inline data/human/hails.txt:2840]:
```suggestion

phrase "friendly civilian"
```

File: data/human/hails.txt [hunk 13]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "Fury"
+    word
+        " "
+    word
+        "that never had"
+    word
+        " "
+    word
+        "stalling engines."
+        "broken batteries."
```
Comment (bene-dictator) [inline data/human/hails.txt:2835]: This one feels off to me.

File: data/human/hails.txt [hunk 14]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+    word
+        " "
+    word
+        "strawberries!"
+        "bananas!"
+        "peaches!"
+        "roses!"
+        "jasmine!"
+        "lavender!"
+phrase "friendly civilian"
```
Comment (bene-dictator) [inline data/human/hails.txt:2865]:
```suggestion

phrase "friendly civilian"
```

File: data/human/hails.txt [hunk 15]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "Have you seen the movie where the" 
+    word
+        " "
+    word
+        "hero"
+        "villain"
+    word
+        " "
+    word
+        "mounted modded twin blasters on a modded Argosy"
```
Comment (bene-dictator) [inline data/human/hails.txt:2876]:
```suggestion
        "mounted overcharged twin blasters on a modded Argosy"
```

File: data/human/hails.txt [hunk 16]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "shoots up a security station with dual space pistols"
+        "dodges quad blaster turrets with a stolen Flivver"
+    word
+        "? So cool."
+phrase "friendly civilian"
```
Comment (bene-dictator) [inline data/human/hails.txt:2881]:
```suggestion

phrase "friendly civilian"
```

File: data/human/hails.txt [hunk 17]
```diff
@@ -2707,7 +2707,189 @@ phrase "friendly civilian"
...
+        "Truthfully, I'm going to be part of the Navy because"
+        "I'm enlisting in the Navy because"
+    word
+        " "
+    word
+        "of their dental insurance."
+        "they'll cover my tuition."
+        "their amazing ship selection."
+        "they offer childcare assistance."
 phrase "friendly navy"
```
Comment (bene-dictator) [inline data/human/hails.txt:2893]:
```suggestion

phrase "friendly navy"
```

**Review by bene-dictator — CHANGES_REQUESTED** [review]
Nice work. You need to replace all your indents with tabulations (tabs) instead of spaces, however. It might be your text editor.

**Comment by blue-four-seven** [comment]
Also I didn't test my writing. I'm not sure how I would go about it.

**Comment by TomGoodIdea** [comment]
Please fill out the PR description template (in some cases you don't have to include all sections, but never leave the unchanged template as your PR's description).

File: data/human/hails.txt [hunk 18]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+	word
+		"stalling engines."
+		"unstable shielding."
+		"overheating fuel cells."
+		"a glitchy HUD."
+		"faulty ship detection."
+
+phrase "friendly civilian"
+	word
+		"Finished cleaning my cargo hold. The"
```
Comment (bene-dictator) [inline data/human/hails.txt:2846]: Removing some trailing whitespaces.
```suggestion
		"Finished cleaning my cargo hold. The"
```

File: data/human/hails.txt [hunk 19]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+
+phrase "friendly civilian"
+	word
+		"Finished cleaning my cargo hold. The" 
+		"Cargo hold is all spick and span. The"
+		"Finished scrubbing the hold. The"
+	word
+		" "
+	word
+		"freshener"
```
Comment (bene-dictator) [inline data/human/hails.txt:2852]:
```suggestion
		"freshener"
```

File: data/human/hails.txt [hunk 20]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		"strawberries!"
+		"bananas!"
+		"peaches!"
+		"roses!"
+		"jasmine!"
+		"lavender!"
+
+phrase "friendly civilian"
+	word
+		"Have you seen the movie where the"
```
Comment (bene-dictator) [inline data/human/hails.txt:2872]: "Holofilm" is used in other hails, may as well use it here too I guess.
```suggestion
		"Have you seen the holofilm where the"
```

File: data/human/hails.txt [hunk 21]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+	word
+		" "
+	word
+		"bad guy"
+		"villain"
+	word
+		" "
+	word
+		"mounts overcharged twin blasters on a modded Argosy"
+		"shoots up a security station with dual space pistols"
```
Comment (bene-dictator) [inline data/human/hails.txt:2882]: "Space pistol" sounds a little cliche, but we already have laser weaponry, so...
```suggestion
		"shoots up a security station with dual laser pistols"
```

File: data/human/hails.txt [hunk 22]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+	word
+		"Frankly, I'm thinking of joining the Navy because"
+		"Truthfully, I'm going to be part of the Navy because"
+		"I'm enlisting in the Navy because"
+	word
+		" "
+	word
+		"of their dental insurance."
+		"they'll cover my tuition."
+		"their amazing ship selection."
```
Comment (bene-dictator) [inline data/human/hails.txt:2897]:
```suggestion
		"of their amazing ship selection."
```

**Review by bene-dictator — CHANGES_REQUESTED** [review]

File: data/human/hails.txt [hunk 23]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		" "
+	word
+		"energy blasters"
+		"particle cannons"
+		"proton guns"
+	word
+		" "
+	word
+		"than"
+		"compared to"
```
Comment (mOctave) [inline data/human/hails.txt:2812]: "compared to" feels a bit stilted here.
```suggestion
```

File: data/human/hails.txt [hunk 24]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		"Shuttle"
+		"Star Barge"
+		"Sparrow"
+		"Heavy Shuttle"
+		"Freighter"
+		"Fury"
+	word
+		" "
+	word
+		"that never had"
```
Comment (mOctave) [inline data/human/hails.txt:2834]: Doesn't really fit. It almost sounds like the issues in the next few lines are things you _want_ your ship to have. I'd rephrase it, or maybe change `It was a"` -> `It was just a"`, and `"that never had"` -> `"but it never had"`

File: data/human/hails.txt [hunk 25]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		"unstable shielding."
+		"overheating fuel cells."
+		"a glitchy HUD."
+		"faulty ship detection."
+
+phrase "friendly civilian"
+	word
+		"Finished cleaning my cargo hold. The" 
+		"Cargo hold is all spick and span. The"
+		"Finished scrubbing the hold. The"
```
Comment (mOctave) [inline data/human/hails.txt:2848]: You could shift all the `. The`s into the next word.

File: data/human/hails.txt [hunk 26]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		"I used makes it smell like"
+	word
+		" "
+	word
+		"strawberries!"
+		"bananas!"
+		"peaches!"
+		"roses!"
+		"jasmine!"
+		"lavender!"
```
Comment (mOctave) [inline data/human/hails.txt:2868]:
```suggestion
	word
		"strawberries"
		"bananas"
		"peaches"
		"roses"
		"jasmine"
		"lavender"
	word
		"."
		"!"
```

File: data/human/hails.txt (hunk 20 above)
Comment (mOctave) [reply data/human/hails.txt:2872]: Could keep both. Movie definitely sounds more natural and less space-opera-y than holofilm does to me, even if holofilm is included in the `visual media` phrase. You could also add a few other alternatives here.

File: data/human/hails.txt [hunk 27]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		" "
+	word
+		"bad guy"
+		"villain"
+	word
+		" "
+	word
+		"mounts overcharged twin blasters on a modded Argosy"
+		"shoots up a security station with dual space pistols"
+		"dodges quad blaster turrets with a stolen Flivver"
```
Comment (mOctave) [inline data/human/hails.txt:2883]:
```suggestion
		"dodges quad blaster turrets in a stolen Flivver"
```

since "with" is a bit unclear (how is the villain doing using the Flivver, etc.)

File: data/human/hails.txt [hunk 28]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		"lavender!"
+
+phrase "friendly civilian"
+	word
+		"Have you seen the movie where the" 
+	word
+		" "
+	word
+		"bad guy"
+		"villain"
```
Comment (mOctave) [inline data/human/hails.txt:2877]: Any reason to limit this to bad guys? I could totally see there being a movie like this where the hero is the one to do these things.

File: data/human/hails.txt [hunk 29]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		"mounts overcharged twin blasters on a modded Argosy"
+		"shoots up a security station with dual space pistols"
+		"dodges quad blaster turrets with a stolen Flivver"
+	word
+		"? So cool."
+
+phrase "friendly civilian"
+	word
+		"Frankly, I'm thinking of joining the Navy because"
+		"Truthfully, I'm going to be part of the Navy because"
```
Comment (mOctave) [inline data/human/hails.txt:2890]: This doesn't really sound like something people would say. How can they guarantee that the Navy will accept them?
```suggestion
		"Honestly, I'm going to join the Navy because"
```

File: data/human/hails.txt (hunk 22 above)
Comment (mOctave) [inline data/human/hails.txt:2897]: Lore-wise, doesn't the Republic do a lot of work with independent captains who don't fly their big fancy ships? I doubt they'd put a merchant captain in command of even a Gunboat/Rainmaker early on. Of course, the merchant might not know that, but still...

Maybe:
```suggestion
		"to see if I can get my hands on one of their warships."
```

File: data/korath/korath weapons.txt
Comment (mOctave) [inline data/korath/korath weapons.txt:1]: I presume this (and the changes to the source) are accidental (maybe caused when resolving merge conflicts)? You should probably revert them.

**Review by mOctave — COMMENTED** [review]
Always nice to see more human content! Overall, things look pretty good, although there's a few slightly confusing parts and some presumably accidental changes to other files.

**Comment by mOctave** [comment]
Oh, also, hails generally don't get tested since there's so many, don't worry about it :)

File: data/human/hails.txt (hunk 22 above)
Comment (blue-four-seven) [reply data/human/hails.txt]: changed to "to see if I can get my hands on one of their warships"

**Comment by blue-four-seven** [comment]
Are the korath weapons changes reverted? It seems like it to me, but I'm not entirely sure.

File: data/human/hails.txt [hunk 30]
```diff
@@ -2708,6 +2708,196 @@ phrase "friendly civilian"
...
+	word
+		"a"
+		"two"
+		"three"
+		"four"
+		"five"
+	word
+		" "
+	word
+		"million credits on "
```
Comment (mOctave) [inline data/human/hails.txt:2732]: If you already have a space after "on," you might as well combine these two words:
```suggestion
	word
		" million credits on "
```

File: data/human/hails.txt [hunk 31]
```diff
@@ -2708,6 +2708,196 @@ phrase "friendly civilian"
...
+		"Pearl"
+		"Glory"
+		"Farseer"
+		"Pilot"
+		"Ada"
+		"Hermes"
+		"Mainsail"
+	word
+		" "
+	word
```
Comment (mOctave) [inline data/human/hails.txt:2750]: Maybe add "playing" here? It makes it sound like each planet has their own version of each game. If that's what you want, it's fine, though.

File: data/human/hails.txt [hunk 32]
```diff
@@ -2708,6 +2708,196 @@ phrase "friendly civilian"
...
+		"Be suspicious of"
+		"Don't trust"
+	word
+		" "
+	word
+		"those space casinos. I blew"
+	word
+		" "
+	word
+		"a"
```
Comment (mOctave) [inline data/human/hails.txt:2724]:
```suggestion
		"half a"
		"a"
```

File: data/human/hails.txt [hunk 33]
```diff
@@ -2708,6 +2708,196 @@ phrase "friendly civilian"
...
+		"Cosmic Chronicles"
+		"Daring Discoveries"
+		"Norn Nebula News"
+		"Southern Rim Spacer News"
+		"Stars and Starships"
+		"Sunracer Stellar Spectacles"
+		"Tarazed Today"
+		"The Galactic Gazette"
+		"The New Moonshake"
+		"The Satellite Sentinel"
```
Comment (mOctave) [inline data/human/hails.txt:2772]:
```suggestion
		"the Galactic Gazette"
		"the New Moonshake"
		"the Satellite Sentinel"
```

File: data/human/hails.txt [hunk 34]
```diff
@@ -2708,6 +2708,196 @@ phrase "friendly civilian"
...
+		"Frankly, I'm thinking of joining the Navy because"
+		"Honestly, I'm going to join the Navy because"
+		"I'm enlisting in the Navy because"
+	word
+		" "
+	word
+		"of their dental insurance."
+		"they'll cover my tuition."
+		"to see if I can get my hands on one of their warships."
+		"they offer childcare assistance."
```
Comment (mOctave) [inline data/human/hails.txt:2899]: Whoops, doing it this way gives a chance of a "because to see" combination.
```suggestion
	word
		"Frankly, I'm thinking of joining the Navy"
		"Honestly, I'm going to join the Navy"
		"I'm enlisting in the Navy"
	word
		" "
	word
		"because of their dental insurance."
		"since they'll cover my tuition."
		"to see if I can get my hands on one of their warships."
		"because they offer childcare assistance."
```

**Review by mOctave — COMMENTED** [review]
Getting close

File: data/human/hails.txt [hunk 35]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+	word
+		"half a"
+		"a"
+		"two"
+		"three"
+		"four"
+		"five"
+	word
+		" million credits on "
+	word
```
Comment (warp-core) [inline data/human/hails.txt:2732]: Is there any particular reason that we have:
```
	word
		" "
	word
		"those space casinos. I blew"
	word
		" "
	word
```
And then also:
```
	word
		" million credits on "
	word
```
?
They appear to be doing the same thing (this part of the result from this phrase should always be this series of words, no other options) but the first puts the leading trailing spaces in their own "word"s while hte second includes them in the "word" with the other words.

File: data/human/hails.txt (hunk 31 above)
Comment (warp-core) [reply data/human/hails.txt:2749]: I don't think "I blew a million credits on Ada poker." Works as a sentence.

File: data/human/hails.txt [hunk 36]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		"Vinci"
+		"Vail"
+		"Calda"
+		"Pearl"
+		"Glory"
+		"Farseer"
+		"Pilot"
+		"Ada"
+		"Hermes"
+		"Mainsail"
```
Comment (warp-core) [inline data/human/hails.txt:2746]: I'm not sure Pilot makes sense on this list.
https://github.com/endless-sky/endless-sky/blob/02e6ded1e7cb9ef7a8e401e71c9accd6133a68b5/data/map%20planets.txt#L3477-L3482
Doesn't appear to be the sort of place where you're likely to find a casino, at least not the sort that these other worlds would have.

Thrall is also a questionable choice:
https://github.com/endless-sky/endless-sky/blob/02e6ded1e7cb9ef7a8e401e71c9accd6133a68b5/data/map%20planets.txt#L4531-L4536
Especially when it's in the same system as Martini, which is clearly the bigger tourist attraction.

And Hermes:
https://github.com/endless-sky/endless-sky/blob/02e6ded1e7cb9ef7a8e401e71c9accd6133a68b5/data/map%20planets.txt#L2120-L2125

Also, what ordering is in use here?

File: data/human/hails.txt [hunk 37]
```diff
@@ -2708,6 +2708,195 @@ phrase "friendly civilian"
...
+		"the Satellite Sentinel"
+	word
+		"? "
+	word
+		"It says that"
+		"It claims that"
+		"It says"
+		"It claims"
+	word
+		" "
```
Comment (warp-core) [inline data/human/hails.txt:2780]:
```suggestion
		"It says"
		"It claims"
	word
		" that "
		" "
```

File: data/human/hails.txt (hunk 36 above)
Comment (blue-four-seven) [reply data/human/hails.txt]: Removed those planets from the list. I alphabetized the list.

File: data/human/hails.txt [hunk 38]
```diff
@@ -2708,6 +2708,167 @@ phrase "friendly civilian"
...
+		"Geminus"
+		"Glory"
+		"Hestia"
+		"Mainsail"
+		"Martini"
+		"Pearl"
+		"Vail"
+		"Vinci"
+	word
+		" "
```
Comment (warp-core) [inline data/human/hails.txt:2741]:
```suggestion
		" playing "
```

File: data/human/hails.txt [hunk 39]
```diff
@@ -2708,6 +2708,167 @@ phrase "friendly civilian"
...
+		"."
+		"!"
+
+phrase "friendly civilian"
+	word
+		"I'm no weapons expert, but it feels better firing "
+	word
+		"energy blasters"
+		"particle cannons"
+		"proton guns"
```
Comment (warp-core) [inline data/human/hails.txt:2791]: References to the specific outfits should be capitalised.
```suggestion
		"Energy Blasters"
		"Particle Cannons"
		"Proton Guns"
```
Also, I'm curious as to why Plasma Cannons are not included in this list?
I think they're much more similar to Paritcle Cannons and Proton Guns than Energy Blasters.
```suggestion
		"Energy Blasters"
		"Particle Cannons"
		"Plasma Cannons"
		"Proton Guns"
```

File: data/human/hails.txt [hunk 40]
```diff
@@ -2708,6 +2708,167 @@ phrase "friendly civilian"
...
+	word
+		" than "
+	word
+		"beam lasers."
+		"heavy lasers."
```
Comment (warp-core) [inline data/human/hails.txt:2796]:
```suggestion
		"Beam Lasers."
		"Heavy Lasers."
```

File: data/human/hails.txt [hunk 41]
```diff
@@ -2708,6 +2708,167 @@ phrase "friendly civilian"
...
+		"Have you seen the holofilm where the "
+	word
+		"good guy"
+		"hero"
+		"bad guy"
+		"villain"
+	word
+		" "
+	word
+		"mounts overcharged twin blasters on a modded Argosy"
```
Comment (warp-core) [inline data/human/hails.txt:2853]:
```suggestion
		"mounts overcharged Twin Blasters on a modded Argosy"
```

File: data/human/hails.txt [hunk 42]
```diff
@@ -2708,6 +2708,167 @@ phrase "friendly civilian"
...
+		"shoots up a security station with dual laser pistols"
+		"dodges quad blaster turrets in a stolen Flivver"
```
Comment (warp-core) [inline data/human/hails.txt:2855]:
```suggestion
		"dodges Quad Blaster Turrets in a stolen Flivver"
```

File: data/human/hails.txt [hunk 43]
```diff
@@ -2708,6 +2708,167 @@ phrase "friendly civilian"
...
+		"Frankly, I'm thinking of joining the Navy"
+		"Honestly, I'm going to join the Navy"
+		"I'm enlisting in the Navy"
+	word
+		" "
+	word
+		"because of their dental insurance."
+		"since they'll cover my tuition."
+		"to see if I can get my hands on one of their warships."
+		"because they offer childcare assistance."
```
Comment (warp-core) [inline data/human/hails.txt:2870]:
```suggestion
		"because of their dental insurance"
		"since they'll cover my tuition"
		"to see if I can get my hands on one of their warships"
		"because they offer childcare assistance"
	word
		"."
```
